### PR TITLE
Add basic docs and remove obsolete files

### PR DESCRIPTION
**Pull request by anehx**
Those docs are very basic for now but will be extendend later on

### CODE REVIEW

File: tests/dummy/app/templates/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Installation
+
+```bash
+ember install ember-caluma-form-builder
+```
+
+# Usage
+
+`ember-caluma-form-builder` is a [routable engine](http://ember-engines.com).
+To include it in your application just mount it in your `app/router.js` and
```
Comment (czosel) [inline tests/dummy/app/templates/docs/index.md]: Maybe also include a snippet for the router?

Comment (anehx) [reply tests/dummy/app/templates/docs/index.md]: Uhm there are exactly two snippets :wink:

File: tests/dummy/app/router.js [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+  docsRoute(this, function() {
+    this.route("usage");
+  });
+
+  this.route("demo", function() {
+    this.mount("ember-caluma-form-builder", {
+      path: "/form-builder",
+      as: "form-builder"
+    });
+  });
```
Comment (czosel) [inline tests/dummy/app/router.js]: You mean this one?

File: tests/dummy/app/templates/docs/index.md (hunk 1 above)
Comment (czosel) [reply tests/dummy/app/templates/docs/index.md]: Duh, nvmd

**Pull request merged**